### PR TITLE
Release

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: src/main/java/com/gmail/chernii/oleksii/list/MyArrayList.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package com.gmail.chernii.oleksii.list;
+
+import java.util.Arrays;
+
+/**
+ * Created by Space on 05.03.2019.
+ */
+public class MyArrayList<E> implements List<E> {
+    private E[] elements;
+    private static final int DEFAULT_LENGTH = 10;
```
Comment (Taniann) [inline src/main/java/com/gmail/chernii/oleksii/list/MyArrayList.java:9]: firstly declare static field, then non-static
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)

File: src/test/java/com/gmail/chernii/oleksii/list/MyArrayListTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import org.junit.Test;
+
+public class MyArrayListTest extends Assert {
+    private List<String> list;
+    private String[] strings = {"one", "two", "three", "four", "five"};
+    private String string = "string";
+
+    @Before
+    public void fillList() {
+        list = new MyArrayList<String>();
```
Comment (Taniann) [inline src/test/java/com/gmail/chernii/oleksii/list/MyArrayListTest.java]: use diamond operator

File: src/test/java/com/gmail/chernii/oleksii/list/MyLinkedListTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import static org.junit.Assert.*;
+
+public class MyLinkedListTest {
+    private List<String> list;
+    private String[] strings = {"one", "two", "three", "four", "five"};
+    private String string = "string";
+
+    @Before
+    public void fillList() {
+        list = new MyLinkedList<String>();
```
Comment (Taniann) [inline src/test/java/com/gmail/chernii/oleksii/list/MyLinkedListTest.java]: thw same about diamond operator

**Review by Taniann — CHANGES_REQUESTED** [review]
good, but make a little fixing please

File: src/test/java/com/gmail/chernii/oleksii/list/MyLinkedListTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package com.gmail.chernii.oleksii.list;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (Taniann) [inline src/test/java/com/gmail/chernii/oleksii/list/MyLinkedListTest.java]: dont use wild card import

File: src/test/java/com/gmail/chernii/oleksii/stack/MyStackTest.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
+package com.gmail.chernii.oleksii.stack;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (Taniann) [inline src/test/java/com/gmail/chernii/oleksii/stack/MyStackTest.java]: the same

**Review by Taniann — APPROVED** [review]
good

**Comment by AlexForJava** [comment]
😉

24 марта 2019, 21:50:30, от "Алексей" <hqq@ukr.net>:

@Taniann approved this pull request.

good

**Pull request merged**